packages/pn-commons/src/components/NotificationDetail/NotificationDetailTable.tsx

```diff
@@ -31,7 +36,7 @@ const NotificationDetailTable = ({ rows }: Props) => (
               flexDirection: { xs: 'column', lg: 'row' },
             }}
           >
-            <TableCell padding="none" sx={{ py: { xs: 0, lg: 1 } }}>
+            <TableCell id="row-label" padding="none" sx={{ py: { xs: 0, lg: 1 } }}>
```

Comment:
@leleOFA non mi sembra corretto, dovrebbe essere un id dinamico dato da row e magari il valore della label o del row id. Essendo in un ciclo andremmo a rompere i vincoli di univocità dell'id